app/main.py

```diff
@@ -16,4 +16,12 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+all_data = [lucky_number, pi, one_is_a_prime_number, name, my_favourite_films,
+            profile_info, marks, collection_of_coins]
+mutable = dict, list, set
+sorted_variables = {"mutable": [], "immutable": []}
+for i in all_data:
```

Comment:
change the name for "i" variable - it is not informative

Reply:
ok, ready